src/util/Constants.ts

```diff
@@ -284,6 +284,65 @@ export const CommandResponses = {
 			.setDescription(`[Direct Link](${avatarURL}).`)
 			.setImage(avatarURL)
 			.setTimestamp();
+	},
+	GUILD_STATS: (guild: Guild) => {
+		const [textChannels, otherType] = guild.channels.cache.partition(ch => ch.type === 'text');
+		const owner = guild.client.users.cache.get(guild.ownerID);
+		return new MessageEmbed()
+			.setColor(SentinelColors.LIGHT_BLUE)
+			.setAuthor(`${guild} information`, guild.iconURL({ dynamic: true }) ?? undefined)
+			.setFooter('Sentinel')
+			.setTimestamp()
+			.addFields({
+				name: 'Guild Overview:',
+				value: [
+					`> Name: ${guild.name}`,
+					`> ID: ${guild.id}`,
+					`> Owner: ${owner ? formatUser(owner) : 'Could not get owner'}`,
+					`> Region: ${guild.region}`,
+					`> Boost Tier: ${guild.premiumTier ? `Tier ${guild.premiumTier}` : 'none'}`,
+					`> Creation Time: ${moment(guild.createdAt).format(DEFAULT_TIME_FORMAT)}`
+				],
+				inline: true
+			}, {
+				name: 'Guild Stats:',
+				value: [
+					`> Member Count: ${guild.memberCount}`,
+					`> Total Roles: ${guild.roles.cache.size}`,
+					`> Total Text Channels: ${textChannels.size}`,
+					`> Total Voice Channels: ${otherType.filter(ch => ch.type === 'voice').size}`,
+					`> Total Emoji Count: ${guild.emojis.cache.size}`,
+					`> Total Boosts: ${guild.premiumSubscriptionCount ? `${guild.premiumSubscriptionCount} boosts` : '0'}`
+				],
+				inline: true
+			});
+	},
+	WHOIS: (member: GuildMember) => {
+		return new MessageEmbed()
+			.setColor(member.displayHexColor || SentinelColors.LIGHT_BLUE)
```

Comment:
`displayHexColor` always returns a string value [See Here](https://github.com/discordjs/discord.js/blob/56e8ef2d38633154540748acb64f4c1305fcc6d3/src/structures/GuildMember.js#L184), perhaps check for truthiness on `GuildMember#roles#color`?